openapi.yaml

```diff
@@ -1333,6 +1333,8 @@ components:
               $ref: "#/components/schemas/Coordinate"
             maps:
               $ref: "#/components/schemas/Maps"
+            comment:
```

Comment:
I dont have a strong oppinion on this, but is this not kind of an `prop`?
Shouldn't it be loated there?

Reply:
`props` is fine as well I think. I also don't have a strong opinion on this, maybe it's a bit more structured this way.